web/views.py

```
@@ -6928,6 +6931,171 @@ def all_study_groups(request):
+
+    # Get content type based on file extension
+    content_type, encoding = mimetypes.guess_type(absolute_path)
+    if not content_type:
+        content_type = "application/octet-stream"
+
+    # Serve the file directly
+    return FileResponse(open(absolute_path, "rb"), content_type=content_type)
+
+
+# API endpoint for dynamic form handling
+def work_type_detail_api(request: HttpRequest, work_type_id: int) -> JsonResponse:
```

Ruff (0.8.2)

Unused function argument: `request`

(ARG001)